### DISCUSSION TITLE
Can I customise my linkfree?

### DISCUSSION THREAD

**Discussion by amelia2802** [EddieHubCommunity/LinkFree#2497] [Q&A] [answered]
I am curious to know that is it possible that I can customise my linkfree profile? Like change the background or change the icon design fonts etc

**Comment by eddiejaoude** [comment, marked as answer]
Yes you can do some customisation like change the icons (new web page to see available icons is in pull request so will be available very soon), more customisation is planned and is coming soon - feel free to create an issue for customisation you would like to see.

Also note you can enable statistics to see how many profile views and link clicks your profile is getting.

You can learn more about what is currently available in the official docs https://linkfree.eddiehub.io/docs

**Reply by amelia2802** [reply]
Sure,. I'll check it. Thank you for your response.

**Reply by eddiejaoude** [reply]
You are welcome 🤓 . Any questions let us know!
